shared/fs/really-delete/container.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import * as FsGen from '../../actions/fs-gen'
+import * as RouteTreeGen from '../../actions/route-tree-gen'
+import * as Container from '../../util/container'
+import ReallyDelete from '.'
+import {anyWaiting} from '../../constants/waiting'
+import * as Types from '../../constants/types/fs'
+import * as Constants from '../../constants/fs'
+
+type OwnProps = Container.RouteProps<{path: Types.Path}, {}>
+
+const mapStateToProps = state => {
+  return {
+    _deleting: (path: Types.Path) => anyWaiting(state, Constants.deleteFolderWaitingKey(path)),
```

Comment:
If you make waitingKey a string (i.e. not dependent on `path`), then this doesn't need to be a function anymore.